api/net/Areas/Subscriber/Controllers/ReportController.cs

```diff
@@ -370,5 +370,31 @@ public async Task<IActionResult> Generate(int id, [FromQuery] bool regenerate =
 
         return new JsonResult(new ReportModel(report, _serializerOptions));
     }
+
+    /// <summary>
+    /// Add the specified content to the current report instance.
+    /// </summary>
+    /// <param name="id"></param>
+    /// <param name="content"></param>
+    /// <returns></returns>
+    /// <exception cref="NotAuthorizedException"></exception>
+    /// <exception cref="NoContentException"></exception>
+    [HttpPost("{id}/content")]
+    [Produces(MediaTypeNames.Application.Json)]
+    [ProducesResponseType(typeof(ReportModel), (int)HttpStatusCode.OK)]
+    [ProducesResponseType(typeof(ErrorResponseModel), (int)HttpStatusCode.BadRequest)]
+    [SwaggerOperation(Tags = new[] { "Report" })]
+    public async Task<IActionResult> AddContentToReportAsync(int id, [FromBody] IEnumerable<ReportInstanceContentModel> content)
```

Comment:
A much better thought out implementation of adding content to a report.  We don't need to update the whole report.  Additionally, this was also updating content...  Everyone needs to be aware of how Entity Framework works when you Add/Update entities.